Assets/Modifiers/Bonus.csv

```diff
@@ -0,0 +1,13 @@
+Bonus Type,What it does,Modifier,Factor,Who,Obsolete
+Ph. D. Thesis in logarithmic time search,Index research shortened for all players,Days,-50,ALL,None
+Perl,All features speed up by 10%,*,0.10,SELF,None
+Java,All features speed up by 10%,*,0.10,SELF,None
+JSP,All features speed up by 10%,*,0.10,SELF,None
+.NET,All features speed up by 12%,*,0.12,SELF,JAVA
```

Comment:
I have been thinking about this for quite awhile, I do not know how I feel about calling the column "Obsolete". I realize it is a bit nit-picky, but as an example Java doesn't make .net obsolete. 

Although I agree with the idea that once you go with java, you shouldn't be able to use .net at the same time.